readme.md

```diff
@@ -13,13 +13,18 @@ cp .env.example .env
 ```shell script
 docker-compose up -d
 ```
+### 3.Install frontend dependencies and build frontend:
+```
+docker-compose exec node npm install
+docker-compose exec node npm run build
```

Comment:
We don't need builded app while we are under development. It can  be removed.
Build app make sense when we want to update app on the beta/live environment, so in our case on github pages.

Reply:
Okay, I will remove this part for now. Thanks!